app/models/vacancy.rb

```diff
@@ -121,7 +121,8 @@ class Vacancy < ApplicationRecord
   delegate :name, to: :school, prefix: true, allow_nil: false
   delegate :geolocation, to: :school, prefix: true, allow_nil: true
 
-  acts_as_gov_uk_date :starts_on, :ends_on, :publish_on, :expires_on
+  acts_as_gov_uk_date :starts_on, :ends_on, :publish_on,
+                      :expires_on, error_clash_behaviour: :omit_gov_uk_date_field_error
```

Comment:
It would be helpful if your commit message explained why you added `error_clash_behaviour: :omit_gov_uk_date_field_error` here 😄